cpp/bench/ann/src/raft/raft_ivf_pq.cu

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "raft_ivf_pq_wrapper.h"
+
+namespace raft::bench::ann {
+template class RaftIvfPQ<float, uint64_t>;
```

Comment:
When we are switching to int8 spcializations, then we should not forget to change these to to int8.

Reply:
I'm wondering if maybe we should just build all supported templates up front? If we're using the pre-compiled specializations properly then it shouldn't really add much time (or space) to the build.